sympy/printing/fcode.py

```diff
@@ -546,6 +574,11 @@ def fcode(expr, assign_to=None, **settings):
         Note that currently the only distinction internally is between
         standards before 95, and those 95 and after. This may change later as
         more features are added.
+    name_mangling : bool, optional
+        If True, then the variables that would become identical in
+        case-insensitive Fortran are mangled by appending different number
+        of `_` at the end. If False, SymPy won't interfere with naming of
```

Comment:
The error message
```
l.85856 of \(_\)
                 at the end. If False, SymPy wonâ€™t interfere with naming of
```
refers to this line. Sphinx is configured to replace the pair of backquotes by `\(` and `\)`, and Latex cannot interpret the resulting `\(_\)`. Double backquotes could probably be used.

Reply:
thanks!